### PR TITLE
HPCC-19561 Document nice-based priorities

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@mckellyln please review
@JamesDeFabia please review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Successful integration testing :  http://10.240.32.243/view/Docs%207.0/job/Docs70-2/

### CODE REVIEW

**Comment by mckellyln** [comment]
@g-pan I think the wording is ok.
Just FYI, I think the initial intended purpose of nice here was to lower the priority of longer running, heavy cpu usage processes, such as Thor, so that user interaction and pre-flight etc. would be more responsive while HPCC jobs were running.

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 1]
```diff
@@ -1215,16 +1215,17 @@ lock=/var/lock/HPCCSystems</programlisting>
       <sect2 id="Using_nice">
         <title>Using nice</title>
 
-        <para>For our purposes <emphasis>nice</emphasis> is the Linux program
-        which invokes scripts and utilities with specified priorities. The
-        priority assigned to a process indicates to the CPU to provide more or
-        less time than to other processes. With a nice value of -20 having the
-        highest priority, and a value of 19 as the lowest priority.</para>
+        <para>The HPCC platform supports <emphasis>nice</emphasis>-based
+        priorities. As nice is the Linux program which invokes scripts and
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: "As nice is the Linux program which invokes scripts and utilities with specified priorities" <--  Is a sentence fragment. Remove the word "As" and it's fine.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
one minor comment

**Review by JamesDeFabia — APPROVED** [review]
Looks good

**Comment by richardkchapman** [comment]
@g-pan Please squash

**Comment by g-pan** [comment]
squashed.

**Pull request merged**